fasthttpadaptor/adaptor_timing_test.go

```diff
@@ -0,0 +1,53 @@
+package fasthttpadaptor
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"testing"
+
+	"github.com/valyala/bytebufferpool"
+	"github.com/valyala/fasthttp"
+)
+
+func BenchmarkAdaptor(b *testing.B) {
```

Comment:
Have you tried running this benchmark before and after your changes to see the improvements?